coral-service/src/main/java/com/linkedin/coral/coralservice/utils/IncrementalUtils.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.coralservice.utils;
+
+import org.apache.calcite.rel.RelNode;
+
+import com.linkedin.coral.hive.hive2rel.HiveToRelConverter;
+import com.linkedin.coral.incremental.RelToIncrementalSqlConverter;
+
+import static com.linkedin.coral.coralservice.utils.CoralProvider.*;
+
+
+public class IncrementalUtils {
+
+  public static String getIncrementalQueryFromUserSql(String query) {
+    RelNode originalNode = new HiveToRelConverter(hiveMetastoreClient).convertSql(query);
+    String incrementalQuery = new RelToIncrementalSqlConverter().convert(originalNode);
```

Comment:
If we move `RelToIncrementalSqlConverter` to the test cases, I think we have similar infra in `coral-service` already, right? We can use it.